app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,20 @@
+class InvitationsController < ApplicationController
+        
+    def create
+        @invitation = current_user.created_invites.build(invitation_params)
```

Comment:
We want the creation of the invitation to anchor more on the invitation itself than the `current_user`. In this case we can simply do:
`@invitation = invitation.create(invitation_params)`

Your `invitation_params` should then include `current_user` 's id. Such that the full parameters you will get from your form are:
`event_id`
`current_user.id`
`invited_id`